prometheus-exporter/DeploymentConfig.yaml

```diff
@@ -0,0 +1,77 @@
+apiVersion: apps.openshift.io/v1
+kind: DeploymentConfig
+metadata:
+  annotations:
+    openshift.io/generated-by: OpenShiftNewApp
```

Comment:
This should not be necessary or at least shouldn't have this value.
I'd remove this annotation